dataset_loaders/config.ini.example

```diff
@@ -1,5 +1,6 @@
 [general]
 datasets_local_path = /the/local/path/where/the/datasets/will/be/copied
+open_permissions = False
```

Comment:
Can you rename it to something more self-explanatory, e.g., `set_local_paths_readable` or `make_local_paths_readable` and add a comment to specify that it will work on Linux (according to https://docs.python.org/2/library/os.html#os.chmod).